File: chess/archive/chess/tile.py
class Tile:
    def __init__(self, r, c):
        self.my_row = r
        self.my_col = c
        self.active = False

    def get_col(self):
        return self.my_col

    def get_row(self):
        return self.my_row

    def make_active(self):
        self.active = True

    def make_inactive(self):
        self.active = False

    def is_active(self):
        return self.active

    def to_string(self):
        return str(self.my_row) + ", " + str(self.my_col)

File: chess/archive/chess-java/src/Tile.java
import javax.swing.*;

public class Tile extends JButton {
    private final int myCol, myRow;
    private boolean isActive;

    public Tile(int row, int col) {
        myRow = row;
        myCol = col;
    }

    public int getCol() {
        return myCol;
    }

    public int getRow() {
        return myRow;
    }

    public void makeActive() {
        isActive = true;
    }

    public void makeInactive() {
        isActive = false;
    }

    public boolean isActive() {
        return isActive;
    }

    public String toString() {
        return "(" + myRow + ", " + myCol + ")";
    }
}
